diff --git a/dist/src/stryker-setup.js b/dist/src/stryker-setup.js
index 8bd1f10..49d5da5 100644
--- a/dist/src/stryker-setup.js
+++ b/dist/src/stryker-setup.js
@@ -63,7 +63,7 @@ function collectTestName({ name, suite, }) {
         nameParts.unshift(currentSuite.name);
         currentSuite = currentSuite.suite;
     }
-    return nameParts.join(' ').trim();
+    return nameParts.join(' > ').trim();
 }
 function toRawTestId(test) {
     return `${test.file?.filepath ?? 'unknown.js'}#${collectTestName(test)}`;
diff --git a/dist/src/test-helpers.js b/dist/src/test-helpers.js
index 6a7a64f..e99659e 100644
--- a/dist/src/test-helpers.js
+++ b/dist/src/test-helpers.js
@@ -8,7 +8,7 @@ export function collectTestName({ name, suite, }) {
         nameParts.unshift(currentSuite.name);
         currentSuite = currentSuite.suite;
     }
-    return nameParts.join(' ').trim();
+    return nameParts.join(' > ').trim();
 }
 export function toRawTestId(test) {
     return `${test.file?.filepath ?? 'unknown.js'}#${collectTestName(test)}`;
diff --git a/src/stryker-setup.ts b/src/stryker-setup.ts
index a37b179..8a36975 100644
--- a/src/stryker-setup.ts
+++ b/src/stryker-setup.ts
@@ -85,5 +85,5 @@ function collectTestName({
     nameParts.unshift(currentSuite.name);
     currentSuite = currentSuite.suite;
   }
-  return nameParts.join(' ').trim();
+  return nameParts.join(' > ').trim();
 }
diff --git a/src/test-helpers.ts b/src/test-helpers.ts
index 66e7d52..fa73369 100644
--- a/src/test-helpers.ts
+++ b/src/test-helpers.ts
@@ -17,5 +17,5 @@ export function collectTestName({
     nameParts.unshift(currentSuite.name);
     currentSuite = currentSuite.suite;
   }
-  return nameParts.join(' ').trim();
+  return nameParts.join(' > ').trim();
 }
